### DISCUSSION TITLE
Single connector for Servos

### DISCUSSION THREAD

**Discussion by ImShomu** [Luisonson/LibreServo_v2#6] [Ideas]
Hello @Luisonson,

Can we have another version of the PCB with a single side of the connector bridge? 
The two-side connector is not suitable for all the development. 
I was wondering if you could just make another version of the PCB with a connector below the motor. 

Best 
Shomu

**Comment by Luisonson** [comment]
Hello @ImShomu ,

You had it, two connectors and a connector on the right or left side 😉https://github.com/Luisonson/LibreServo_v2/tree/master/PCB/Gerbers/LibreServo%20v2.3.1

**Reply by SoumyaMaitra** [reply]
Sorry for any misunderstanding. 

I wanted to have a PCB with one wing for one set of connectors but below the motor. 
I was not asking for the left or right wing. 
As I am not an expert in PCB making or soldering the SMD components wanted to take your help in making.

If you have time please let me know if you can help me in it.
